modules/tensor_mechanics/doc/content/syntax/Modules/TensorMechanics/Master/index.md

```diff
@@ -25,6 +25,7 @@ The Tensor Mechanics `Master` Action is used to construct the kernels, displacem
 | | [ComputeSmallStrain](/ComputeSmallStrain.md) or [ComputePlaneSmallStrain](/ComputePlaneSmallStrain.md) or [ComputeAxisymmetric1DSmallStrain](/ComputeAxisymmetric1DSmallStrain.md) or [ComputeAxisymmetricRZSmallStrain](/ComputeAxisymmetricRZSmallStrain.md) | |
 | | [ComputeIncrementalSmallStrain](/ComputeIncrementalSmallStrain.md) or [ComputePlaneIncrementalStrain](/ComputePlaneIncrementalStrain.md) or [ComputeAxisymmetric1DIncrementalStrain](/ComputeAxisymmetric1DIncrementalStrain.md) or [ComputeAxisymmetricRZIncrementalStrain](/ComputeAxisymmetricRZIncrementalStrain.md) |`incremental` : boolean for using a incremental strain formulation |
 | Add AuxVariables and AuxKernels for various tensor component and quantity outputs | [AuxVariables](/AuxVariables/index.md) and [RankTwoAux](/RankTwoAux.md) or [RankTwoScalarAux](/RankTwoScalarAux.md) or [RankFourAux](/RankFourAux.md) | `generate_output`: a string of the quantities to add |
+| Add GlobalStrain contribution to the strain calculation | None, [GlobalStrain](/GlobalStrain/index.md) calculation is set up separately | `global_strain`: name of the material property that calculates the global strain tensor |
```

Comment:
What do you think about `Couples to the [GlobalStrain](/GlobalStrain.md) optional calculation` ? I think that might work nicely for the eigenstrain contributions too?